plugins/auth-backend/src/providers/cloudflare-access/provider.test.ts

```diff
@@ -138,12 +138,28 @@ describe('CloudflareAccessAuthProvider', () => {
   const provider = new CloudflareAccessAuthProvider({
     teamName: 'foobar',
     resolverContext: {} as AuthResolverContext,
-    authHandler: async ({ claims }) => ({
-      profile: {
-        email: claims.email,
-      },
-    }),
-    signInResolver: async () => {
+    authHandler: async result => {
+      expect(result).toEqual(
```

Comment:
It's a bit awkward to put expectations in these test fixtures, but I wasn't sure of a cleaner way to accomplish this type of verification. Open to any suggestions :)